src/follower.py

```diff
@@ -117,32 +128,47 @@ def reset_camera(self):
     will return False.
     """
     def detect(self):
-        obj_data = self._tag.detect()
-        detected = False
-
-        if obj_data:
-            detected = True
-            self._distance = obj_data['z']
-            self._turn_angle = np.degrees(obj_data['direction'])
-            self._decision = obj_data['decision']
+        tag_data = self._tag.detect()
```

Comment:
Why'd you rewrite all of this? I have a few comments about it:

1. Don't change the variable name here for no reason. It's detecting an object. That object just so happens to be a tag. Additionally, this was reviewed and merged with no complaints. Please revert this back to obj_data.
2. You changed the entire if statement below, but only the parameter of the if statement needed to be changed (if the name change was going through). This blows up the line count for no reason.
3. There's a reason I didn't return tag_data in the if statement. Now the function returns two different data types. That's confusing and terrible for a public facing function. The way I had it, it returned a boolean 100% of the time and only had one exit point. This is more maintainable and testable than returning the tag data. There's also no benefit to returning the tag_data here. We don't do anything with it. There's a reason the variables are being set in this function.
4. IF this was to be changed, it should be in its own pull request. It has nothing to do with camera panning.

Please get rid of these changes.